src/Propel/Generator/Command/Helper/ConsoleHelper3.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Propel\Generator\Command\Helper;
+
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class ConsoleHelper3 extends QuestionHelper implements ConsoleHelperInterface
+{
+    /** @var InputInterface $input */
+    protected $input;
+
+    /** @var OutputInterface $output */
+    protected $output;
+
+    public function __construct(InputInterface $input, OutputInterface $output)
+    {
+        $this->input = $input;
+        $this->output = $output;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function askQuestion($question, $default = null, array $autocomplete = null)
+    {
+        $question = new Question($this->formatQuestion($question, $default), $default);
+        $question->setAutocompleterValues($autocomplete);
+
+        return parent::ask($this->input, $this->output, $question);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function askHiddenResponse($question, $fallback = true)
+    {
+        $question = new Question($this->formatQuestion($question));
+        $question->setHidden(true);
+        $question->setHiddenFallback($fallback);
+
+        return parent::ask($this->input, $this->output, $question);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function writeSection($text)
+    {
+        $this->output->writeln([
+            '',
+            $text,
+        ]);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function writeBlock($text, $style = 'bg=blue;fg=white')
+    {
+        /** @var FormatterHelper $formatter */
+        $formatter = $this->getHelperSet()->get('formatter');
+        $block = $formatter->formatBlock($text, $style, true);
+
+        $this->writeSection($block);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function writeSummary($items)
+    {
+        $this->output->writeln('');
+        foreach ($items as $name => $value) {
+            $this->output->writeln(sprintf('<info>%s</info>: <comment>%s</comment>', $name, $value));
+        }
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function select($question, $choices, $default = null, $attempts = false, $errorMessage = 'Value "%s" is invalid', $multiselect = false)
+    {
+        $question = new ChoiceQuestion($this->formatQuestion($question, $default), $choices, $default);
+        $attempts && $question->setMaxAttempts($attempts);
```

Comment:
@phlopsi thanks for a second opinion on this.

@SCIF as @phlopsi has pointed out it's not a common used practice in the php world, although the style is used in other programming languages. You will agree that code should be as readable as possible, and sometimes this means following conventions even when it not follows your personal style. I know this too well, sometimes following the convention sucks, but for open source software and software maintained by more than a single person in general, it is necessary to adapt ones coding style to the project. Remember: Your style is not wrong, it's just uncommon.

So let's just fix this and we're ready to merge :rocket: